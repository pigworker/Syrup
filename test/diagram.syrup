not(<Bit>) -> <Bit>
not(x) = nand(x, x)

-- display not

or(<Bit>, <Bit>) -> <Bit>
or(X,Y) = nand(!X,!Y)

--display or
--display [not] or