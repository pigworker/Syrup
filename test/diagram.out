Circuit not is defined.

Displaying not:
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<!-- Generated by graphviz version 2.43.0 (0)
 -->
<!-- Title: whitebox Pages: 1 -->
<svg width="114pt" height="334pt"
 viewBox="0.00 0.00 114.00 334.00" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="graph0" class="graph" transform="scale(1 1) rotate(0) translate(4 330)">
<title>whitebox</title>
<polygon fill="white" stroke="transparent" points="-4,4 -4,-330 110,-330 110,4 -4,4"/>
<g id="clust1" class="cluster">
<title>cluster_circuit__GATE_not_3</title>
</g>
<!-- GATE_not_3__INPUTS -->
<g id="node1" class="node">
<title>GATE_not_3__INPUTS</title>
<text text-anchor="start" x="48" y="-294" font-family="Times,serif" font-size="20.00">x</text>
</g>
<!-- COPY_3__1 -->
<g id="node3" class="node">
<title>COPY_3__1</title>
<polygon fill="skyblue" stroke="transparent" points="58,-221 48,-221 48,-214 58,-214 58,-221"/>
</g>
<!-- GATE_not_3__INPUTS&#45;&gt;COPY_3__1 -->
<g id="edge4" class="edge">
<title>GATE_not_3__INPUTS:x&#45;&gt;COPY_3__1:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M53,-285.97C53,-271.07 53,-244.91 53,-222.5"/>
<text text-anchor="middle" x="55" y="-242.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- GATE_nand_3__0__0 -->
<g id="node2" class="node">
<title>GATE_nand_3__0__0</title>
<text text-anchor="start" x="37.5" y="-146.4" font-family="Times,serif" font-size="7.00">X</text>
<text text-anchor="start" x="63" y="-146.4" font-family="Times,serif" font-size="7.00">Y</text>
<text text-anchor="start" x="34" y="-120" font-family="Times,serif" font-size="20.00">nand</text>
<text text-anchor="start" x="38" y="-100.4" font-family="Times,serif" font-size="7.00">Z</text>
<polygon fill="none" stroke="black" points="24.5,-91 24.5,-159 81.5,-159 81.5,-91 24.5,-91"/>
</g>
<!-- GATE_not_3__OUTPUTS -->
<g id="node4" class="node">
<title>GATE_not_3__OUTPUTS</title>
</g>
<!-- GATE_nand_3__0__0&#45;&gt;GATE_not_3__OUTPUTS -->
<g id="edge3" class="edge">
<title>GATE_nand_3__0__0:s&#45;&gt;GATE_not_3__OUTPUTS:__VIRTUAL__2</title>
<path fill="none" stroke="black" stroke-width="2" d="M40,-95C40,-69.17 40,-38.41 40,-25.36"/>
<polygon fill="black" stroke="black" stroke-width="2" points="41.75,-25.07 40,-20.07 38.25,-25.07 41.75,-25.07"/>
<text text-anchor="middle" x="42" y="-57.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- COPY_3__1&#45;&gt;GATE_nand_3__0__0 -->
<g id="edge1" class="edge">
<title>COPY_3__1:s&#45;&gt;GATE_nand_3__0__0:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M51,-214C51,-205.8 48.58,-204.04 47,-196 43.8,-179.7 40.7,-174.52 40.1,-160.22"/>
<polygon fill="black" stroke="black" stroke-width="2" points="41.85,-159.96 40,-155 38.35,-160.03 41.85,-159.96"/>
<text text-anchor="middle" x="49" y="-184.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- COPY_3__1&#45;&gt;GATE_nand_3__0__0 -->
<g id="edge2" class="edge">
<title>COPY_3__1:s&#45;&gt;GATE_nand_3__0__0:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M55,-214C55,-189.1 64.59,-182.52 65.86,-160.05"/>
<polygon fill="black" stroke="black" stroke-width="2" points="67.61,-160.05 66,-155 64.11,-159.95 67.61,-160.05"/>
<text text-anchor="middle" x="63" y="-184.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
</g>
</svg>

Circuit or is defined.

Displaying or:
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<!-- Generated by graphviz version 2.43.0 (0)
 -->
<!-- Title: whitebox Pages: 1 -->
<svg width="165pt" height="395pt"
 viewBox="0.00 0.00 165.00 395.00" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="graph0" class="graph" transform="scale(1 1) rotate(0) translate(4 391)">
<title>whitebox</title>
<polygon fill="white" stroke="transparent" points="-4,4 -4,-391 161,-391 161,4 -4,4"/>
<g id="clust1" class="cluster">
<title>cluster_circuit__GATE_or_4</title>
</g>
<!-- GATE_or_4__INPUTS -->
<g id="node1" class="node">
<title>GATE_or_4__INPUTS</title>
<text text-anchor="start" x="67" y="-355" font-family="Times,serif" font-size="20.00">X</text>
<text text-anchor="start" x="96" y="-355" font-family="Times,serif" font-size="20.00">Y</text>
</g>
<!-- GATE_not_4__0__3 -->
<g id="node2" class="node">
<title>GATE_not_4__0__3</title>
<text text-anchor="start" x="111.5" y="-265.4" font-family="Times,serif" font-size="7.00">x</text>
<text text-anchor="start" x="101" y="-239" font-family="Times,serif" font-size="20.00">not</text>
<polygon fill="none" stroke="black" points="93.5,-218 93.5,-278 132.5,-278 132.5,-218 93.5,-218"/>
</g>
<!-- GATE_or_4__INPUTS&#45;&gt;GATE_not_4__0__3 -->
<g id="edge5" class="edge">
<title>GATE_or_4__INPUTS:Y&#45;&gt;GATE_not_4__0__3:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M104,-346C104,-315.46 112.94,-307.19 113.91,-279.08"/>
<polygon fill="black" stroke="black" stroke-width="2" points="115.66,-279.03 114,-274 112.17,-278.97 115.66,-279.03"/>
<text text-anchor="middle" x="113" y="-303.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- GATE_not_4__1__3 -->
<g id="node3" class="node">
<title>GATE_not_4__1__3</title>
<text text-anchor="start" x="42.5" y="-265.4" font-family="Times,serif" font-size="7.00">x</text>
<text text-anchor="start" x="32" y="-239" font-family="Times,serif" font-size="20.00">not</text>
<polygon fill="none" stroke="black" points="24.5,-218 24.5,-278 63.5,-278 63.5,-218 24.5,-218"/>
</g>
<!-- GATE_or_4__INPUTS&#45;&gt;GATE_not_4__1__3 -->
<g id="edge4" class="edge">
<title>GATE_or_4__INPUTS:X&#45;&gt;GATE_not_4__1__3:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M74,-346C74,-313.25 47.87,-308.88 45.22,-279.02"/>
<polygon fill="black" stroke="black" stroke-width="2" points="46.96,-278.92 45,-274 43.47,-279.07 46.96,-278.92"/>
<text text-anchor="middle" x="64" y="-303.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- GATE_nand_4__2__0 -->
<g id="node4" class="node">
<title>GATE_nand_4__2__0</title>
<text text-anchor="start" x="74.5" y="-146.4" font-family="Times,serif" font-size="7.00">X</text>
<text text-anchor="start" x="100" y="-146.4" font-family="Times,serif" font-size="7.00">Y</text>
<text text-anchor="start" x="71" y="-120" font-family="Times,serif" font-size="20.00">nand</text>
<text text-anchor="start" x="75" y="-100.4" font-family="Times,serif" font-size="7.00">Z</text>
<polygon fill="none" stroke="black" points="61.5,-91 61.5,-159 118.5,-159 118.5,-91 61.5,-91"/>
</g>
<!-- GATE_not_4__0__3&#45;&gt;GATE_nand_4__2__0 -->
<g id="edge2" class="edge">
<title>GATE_not_4__0__3:s&#45;&gt;GATE_nand_4__2__0:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M114,-222C114,-193.59 104.25,-186.04 103.11,-160.07"/>
<polygon fill="black" stroke="black" stroke-width="2" points="104.86,-159.96 103,-155 101.36,-160.04 104.86,-159.96"/>
<text text-anchor="middle" x="112" y="-184.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- GATE_not_4__1__3&#45;&gt;GATE_nand_4__2__0 -->
<g id="edge3" class="edge">
<title>GATE_not_4__1__3:s&#45;&gt;GATE_nand_4__2__0:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M45,-222C45,-190.8 73.6,-188.2 76.72,-160.14"/>
<polygon fill="black" stroke="black" stroke-width="2" points="78.48,-160.09 77,-155 74.98,-159.9 78.48,-160.09"/>
<text text-anchor="middle" x="69" y="-184.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- GATE_or_4__OUTPUTS -->
<g id="node5" class="node">
<title>GATE_or_4__OUTPUTS</title>
</g>
<!-- GATE_nand_4__2__0&#45;&gt;GATE_or_4__OUTPUTS -->
<g id="edge1" class="edge">
<title>GATE_nand_4__2__0:s&#45;&gt;GATE_or_4__OUTPUTS:__VIRTUAL__0</title>
<path fill="none" stroke="black" stroke-width="2" d="M77,-95C77,-69.17 77,-38.41 77,-25.36"/>
<polygon fill="black" stroke="black" stroke-width="2" points="78.75,-25.07 77,-20.07 75.25,-25.07 78.75,-25.07"/>
<text text-anchor="middle" x="79" y="-57.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
</g>
</svg>

Displaying or (with not unfolded):
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN"
 "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<!-- Generated by graphviz version 2.43.0 (0)
 -->
<!-- Title: whitebox Pages: 1 -->
<svg width="228pt" height="594pt"
 viewBox="0.00 0.00 228.00 594.00" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="graph0" class="graph" transform="scale(1 1) rotate(0) translate(4 590)">
<title>whitebox</title>
<polygon fill="white" stroke="transparent" points="-4,4 -4,-590 224,-590 224,4 -4,4"/>
<g id="clust1" class="cluster">
<title>cluster_circuit__GATE_or_4</title>
</g>
<g id="clust2" class="cluster">
<title>cluster_circuit__GATE_not_4__0__3</title>
<polygon fill="none" stroke="black" points="114,-257 114,-440 204,-440 204,-257 114,-257"/>
<polygon fill="none" stroke="black" points="124,-412.5 124,-433.5 148,-433.5 148,-412.5 124,-412.5"/>
<text text-anchor="start" x="127" y="-419.3" font-family="Times,serif" font-size="14.00">not</text>
</g>
<g id="clust5" class="cluster">
<title>cluster_circuit__GATE_not_4__1__3</title>
<polygon fill="none" stroke="black" points="16,-257 16,-440 106,-440 106,-257 16,-257"/>
<polygon fill="none" stroke="black" points="26,-412.5 26,-433.5 50,-433.5 50,-412.5 26,-412.5"/>
<text text-anchor="start" x="29" y="-419.3" font-family="Times,serif" font-size="14.00">not</text>
</g>
<!-- GATE_or_4__INPUTS -->
<g id="node1" class="node">
<title>GATE_or_4__INPUTS</title>
<text text-anchor="start" x="86" y="-554" font-family="Times,serif" font-size="20.00">X</text>
<text text-anchor="start" x="115" y="-554" font-family="Times,serif" font-size="20.00">Y</text>
</g>
<!-- NODE_x_4__0__3 -->
<g id="node3" class="node">
<title>NODE_x_4__0__3</title>
<ellipse fill="black" stroke="black" cx="124" cy="-481" rx="0" ry="0"/>
</g>
<!-- GATE_or_4__INPUTS&#45;&gt;NODE_x_4__0__3 -->
<g id="edge11" class="edge">
<title>GATE_or_4__INPUTS:Y&#45;&gt;NODE_x_4__0__3:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M123,-545C123,-517 124,-510 124,-482"/>
<text text-anchor="middle" x="125" y="-502.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- NODE_x_4__1__3 -->
<g id="node7" class="node">
<title>NODE_x_4__1__3</title>
<ellipse fill="black" stroke="black" cx="84" cy="-481" rx="0" ry="0"/>
</g>
<!-- GATE_or_4__INPUTS&#45;&gt;NODE_x_4__1__3 -->
<g id="edge10" class="edge">
<title>GATE_or_4__INPUTS:X&#45;&gt;NODE_x_4__1__3:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M93,-545C93,-516.72 84,-510.28 84,-482"/>
<text text-anchor="middle" x="90" y="-502.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- NODE___VIRTUAL__2_4__0__3 -->
<g id="node2" class="node">
<title>NODE___VIRTUAL__2_4__0__3</title>
<ellipse fill="black" stroke="black" cx="136" cy="-214" rx="0" ry="0"/>
</g>
<!-- GATE_nand_4__2__0 -->
<g id="node10" class="node">
<title>GATE_nand_4__2__0</title>
<text text-anchor="start" x="104.5" y="-146.4" font-family="Times,serif" font-size="7.00">X</text>
<text text-anchor="start" x="130" y="-146.4" font-family="Times,serif" font-size="7.00">Y</text>
<text text-anchor="start" x="101" y="-120" font-family="Times,serif" font-size="20.00">nand</text>
<text text-anchor="start" x="105" y="-100.4" font-family="Times,serif" font-size="7.00">Z</text>
<polygon fill="none" stroke="black" points="91.5,-91 91.5,-159 148.5,-159 148.5,-91 91.5,-91"/>
</g>
<!-- NODE___VIRTUAL__2_4__0__3&#45;&gt;GATE_nand_4__2__0 -->
<g id="edge12" class="edge">
<title>NODE___VIRTUAL__2_4__0__3:s&#45;&gt;GATE_nand_4__2__0:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M136,-213C136,-189 133.41,-181.8 133.04,-160.17"/>
<polygon fill="black" stroke="black" stroke-width="2" points="134.79,-159.99 133,-155 131.29,-160.01 134.79,-159.99"/>
<text text-anchor="middle" x="137" y="-184.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- COPY_4__0__3__1 -->
<g id="node5" class="node">
<title>COPY_4__0__3__1</title>
<polygon fill="skyblue" stroke="transparent" points="153,-399 143,-399 143,-392 153,-392 153,-399"/>
</g>
<!-- NODE_x_4__0__3&#45;&gt;COPY_4__0__3__1 -->
<g id="edge4" class="edge">
<title>NODE_x_4__0__3&#45;&gt;COPY_4__0__3__1:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M124.07,-479.88C125.48,-477.34 148,-436.25 148,-400.5"/>
<text text-anchor="middle" x="139" y="-451.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- GATE_nand_4__0__3__0__0 -->
<g id="node4" class="node">
<title>GATE_nand_4__0__3__0__0</title>
<text text-anchor="start" x="143.5" y="-324.4" font-family="Times,serif" font-size="7.00">X</text>
<text text-anchor="start" x="169" y="-324.4" font-family="Times,serif" font-size="7.00">Y</text>
<text text-anchor="start" x="140" y="-298" font-family="Times,serif" font-size="20.00">nand</text>
<text text-anchor="start" x="144" y="-278.4" font-family="Times,serif" font-size="7.00">Z</text>
<polygon fill="none" stroke="black" points="130.5,-269 130.5,-337 187.5,-337 187.5,-269 130.5,-269"/>
</g>
<!-- GATE_nand_4__0__3__0__0&#45;&gt;NODE___VIRTUAL__2_4__0__3 -->
<g id="edge3" class="edge">
<title>GATE_nand_4__0__3__0__0:s&#45;&gt;NODE___VIRTUAL__2_4__0__3</title>
<path fill="none" stroke="black" stroke-width="2" d="M146,-273C146,-247.66 136.62,-216.98 136.03,-215.09"/>
<text text-anchor="middle" x="145" y="-235.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- COPY_4__0__3__1&#45;&gt;GATE_nand_4__0__3__0__0 -->
<g id="edge1" class="edge">
<title>COPY_4__0__3__1:s&#45;&gt;GATE_nand_4__0__3__0__0:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M146,-392C146,-367.62 146,-360.24 146,-338.25"/>
<polygon fill="black" stroke="black" stroke-width="2" points="147.75,-338 146,-333 144.25,-338 147.75,-338"/>
<text text-anchor="middle" x="148" y="-362.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- COPY_4__0__3__1&#45;&gt;GATE_nand_4__0__3__0__0 -->
<g id="edge2" class="edge">
<title>COPY_4__0__3__1:s&#45;&gt;GATE_nand_4__0__3__0__0:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M150,-392C150,-365.87 169.18,-361.5 171.72,-338.26"/>
<polygon fill="black" stroke="black" stroke-width="2" points="173.48,-338.09 172,-333 169.99,-337.9 173.48,-338.09"/>
<text text-anchor="middle" x="164" y="-362.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- NODE___VIRTUAL__2_4__1__3 -->
<g id="node6" class="node">
<title>NODE___VIRTUAL__2_4__1__3</title>
<ellipse fill="black" stroke="black" cx="53" cy="-214" rx="0" ry="0"/>
</g>
<!-- NODE___VIRTUAL__2_4__1__3&#45;&gt;GATE_nand_4__2__0 -->
<g id="edge13" class="edge">
<title>NODE___VIRTUAL__2_4__1__3:s&#45;&gt;GATE_nand_4__2__0:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M53,-213C53,-179.57 101.65,-189.08 106.6,-160.01"/>
<polygon fill="black" stroke="black" stroke-width="2" points="108.34,-160.12 107,-155 104.85,-159.84 108.34,-160.12"/>
<text text-anchor="middle" x="88" y="-184.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- COPY_4__1__3__1 -->
<g id="node9" class="node">
<title>COPY_4__1__3__1</title>
<polygon fill="skyblue" stroke="transparent" points="77,-399 67,-399 67,-392 77,-392 77,-399"/>
</g>
<!-- NODE_x_4__1__3&#45;&gt;COPY_4__1__3__1 -->
<g id="edge8" class="edge">
<title>NODE_x_4__1__3&#45;&gt;COPY_4__1__3__1:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M83.97,-479.87C83.26,-477.27 72,-435.12 72,-400.5"/>
<text text-anchor="middle" x="81" y="-451.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- GATE_nand_4__1__3__0__0 -->
<g id="node8" class="node">
<title>GATE_nand_4__1__3__0__0</title>
<text text-anchor="start" x="45.5" y="-324.4" font-family="Times,serif" font-size="7.00">X</text>
<text text-anchor="start" x="71" y="-324.4" font-family="Times,serif" font-size="7.00">Y</text>
<text text-anchor="start" x="42" y="-298" font-family="Times,serif" font-size="20.00">nand</text>
<text text-anchor="start" x="46" y="-278.4" font-family="Times,serif" font-size="7.00">Z</text>
<polygon fill="none" stroke="black" points="32.5,-269 32.5,-337 89.5,-337 89.5,-269 32.5,-269"/>
</g>
<!-- GATE_nand_4__1__3__0__0&#45;&gt;NODE___VIRTUAL__2_4__1__3 -->
<g id="edge7" class="edge">
<title>GATE_nand_4__1__3__0__0:s&#45;&gt;NODE___VIRTUAL__2_4__1__3</title>
<path fill="none" stroke="black" stroke-width="2" d="M48,-273C48,-247.94 52.69,-217 52.99,-215.09"/>
<text text-anchor="middle" x="52" y="-235.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- COPY_4__1__3__1&#45;&gt;GATE_nand_4__1__3__0__0 -->
<g id="edge5" class="edge">
<title>COPY_4__1__3__1:s&#45;&gt;GATE_nand_4__1__3__0__0:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M70,-392C70,-381.87 60.2,-383.23 56,-374 49.09,-358.82 48.14,-352.81 48.02,-338.28"/>
<polygon fill="black" stroke="black" stroke-width="2" points="49.77,-337.99 48,-333 46.27,-338.01 49.77,-337.99"/>
<text text-anchor="middle" x="58" y="-362.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- COPY_4__1__3__1&#45;&gt;GATE_nand_4__1__3__0__0 -->
<g id="edge6" class="edge">
<title>COPY_4__1__3__1:s&#45;&gt;GATE_nand_4__1__3__0__0:n</title>
<path fill="none" stroke="black" stroke-width="2" d="M74,-392C74,-367.62 74,-360.24 74,-338.25"/>
<polygon fill="black" stroke="black" stroke-width="2" points="75.75,-338 74,-333 72.25,-338 75.75,-338"/>
<text text-anchor="middle" x="76" y="-362.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
<!-- GATE_or_4__OUTPUTS -->
<g id="node11" class="node">
<title>GATE_or_4__OUTPUTS</title>
</g>
<!-- GATE_nand_4__2__0&#45;&gt;GATE_or_4__OUTPUTS -->
<g id="edge9" class="edge">
<title>GATE_nand_4__2__0:s&#45;&gt;GATE_or_4__OUTPUTS:__VIRTUAL__0</title>
<path fill="none" stroke="black" stroke-width="2" d="M107,-95C107,-69.17 107,-38.41 107,-25.36"/>
<polygon fill="black" stroke="black" stroke-width="2" points="108.75,-25.07 107,-20.07 105.25,-25.07 108.75,-25.07"/>
<text text-anchor="middle" x="109" y="-57.8" font-family="Times,serif" font-size="14.00"> </text>
</g>
</g>
</svg>
